.github/workflows/release-please.yaml

```diff
@@ -0,0 +1,104 @@
+# #############################################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# #############################################################################
+
+on:
+  push:
+    branches:
+      - main
+permissions:
+    contents: write
+    pull-requests: write
+name: Create release-automation GitHub Release
+env:
+  APP_NAME: 'release-automation'
+
+jobs:
+    prepare-release:
+        runs-on: ubuntu-latest
+        outputs:
+          upload_url: ${{ steps.release.outputs[format('{0}--upload_url', env.APP_NAME)] }}
+          release_created: ${{ steps.release.outputs[format('{0}--release_created', env.APP_NAME)] }}
+          version: ${{ steps.release.outputs[format('{0}--major',env.APP_NAME)] }}.${{ steps.release.outputs[format('{0}--minor', env.APP_NAME)] }}.${{ steps.release.outputs[format('{0}--patch', env.APP_NAME)] }}
+        steps:
+        - uses: google-github-actions/release-please-action@v4
+          name: Prepare release
+          id: release
+          with:
+            path: ${{ env.APP_NAME }}
+            release-type: go
+        - uses: actions/checkout@v4
+        - name: Tag major and minor versions
+          if: ${{ steps.release.outputs.release-automation--release_created }}
+          run: |
+            git config user.name github-actions[bot]
```

Comment:
Hi @tomaszbarwicki, 

could you please explain, why this manual tagging is necessary, when using the release-please action?
You can have a look at this example: https://github.com/catenax-ng/release-automation-playground/blob/main/.github/workflows/release.yaml#L24

There is almost no manual step necessary except the building an uploading. 
I think having this in a single job is maybe also easier than having two jobs and then needing to set outputs

Reply:
Hi @SebastianBezold,
the reason of creating tags is because we want as I believe to use release-automation as GitHub action, replacing following: https://github.com/eclipse-tractusx/sig-infra/blob/main/.github/workflows/reusable-quality-checks.yaml therefore we need to create appropriate tags as suggested by 
https://github.com/google-github-actions/release-please-action#creating-majorminor-tags

Reply:
Ah ok I see! makes sense in our case of releasing the action as well in the same repo. 
I think i saw a feature of release-please, that would allow us to replace the version number of the action with the outputs of release please too. Maybe this would be a good next step then, since it feels odd, that we only need that for the action.yaml file, which we might forget to increment in the PR.
In such cases, we would tag the commit, but would create a following commit, that then bumps the version used in the action. Could lead to the action being behind one version always :)